### ISSUE TITLE
For mixed-precision training, does FSDP2 also need `amp.grad_scaler.GradScaler` ? or is FSDP2 already handled?

### ISSUE THREAD

**Issue by EquationWalker** [pytorch/pytorch#163519]
In mixed-precision training of DDP, `amp.grad_scaler.GradScaler`  is needed to dynamically scale the loss, my question is does FSDP2 also need `amp.grad_scaler.GradScaler` ? or is FSDP2 already handled?

cc @H-Huang @awgu @wanchaol @fegin @fduwjj @wz337 @wconstab @d4l3k @pragupta @ezyang @msaroufim @dcci

**Comment by janeyx99** [comment]
cc @weifengpy
